src/Vlingo.Actors/Plugin/Logging/Console/ConsoleLogger.cs

```diff
@@ -34,16 +35,53 @@ public void Close()
         {
         }
 
-        public void Log(string message)
+        public void Debug(string message) => Log(message, "Debug");
+
+        public void Debug(string message, params object[] args) => Log(message, "Debug", args);
```

Comment:
not a big deal but wouldn't be less error prone with enum ?

Reply:
Maybe. `Log()` is just a refactored private method. I think we are fine here.